messages/sdr.md

```diff
@@ -22,6 +22,10 @@ Could not find parent type for %s (%s)
 
 Component conversion failed: %s
 
+# error_invalid_test_level
+
+TestLevel cannot be '%s' unless API version is %s or later
```

Comment:
I imagine this message needs to go through some kind of review from a tech writer?